runtime/compiler/optimizer/J9TransformUtil.cpp

```diff
@@ -40,6 +40,21 @@
 #include "j9.h"
 #include "optimizer/OMROptimization_inlines.hpp"
 
+/**
+ * Walks the TR_RegionStructure counting loops to get the nesting depth of the block
+ */ 
+int32_t J9::TransformUtil::getLoopNestingDepth(TR::Compilation *comp, TR::Block *block)
```

Comment:
is the getNestingDepth on TR_Structure not enough?

Reply:
I tried calling `region->getNestingDepth()` , it returns `0` . Checking in more detail, it just returns `_nestingDepth` set in structure. Going in more detail, I see that a call to setNestingDepths(...) sets the nesting depth till it hits a region which is not `naturalLoop`.
I was expecting by this time in compilation it should be set.